packages/hash/integration/src/mockData/index.ts

```diff
@@ -731,54 +747,46 @@ void (async () => {
           createdByAccountId: user.entityId,
           properties: {
             contents: [
-              {
-                entityId: results.get("b1")!.entityId,
-                accountId: results.get("b1")!.accountId,
-              },
-              {
-                entityId: results.get("b9")!.entityId,
-                accountId: results.get("b9")!.accountId,
-              },
-              {
-                entityId: results.get("b11")!.entityId,
-                accountId: results.get("b11")!.accountId,
-              },
-              {
-                entityId: results.get("b2")!.entityId,
-                accountId: results.get("b2")!.accountId,
-              },
-              {
-                entityId: results.get("b3")!.entityId,
-                accountId: results.get("b3")!.accountId,
-              },
-              {
-                entityId: results.get("b10")!.entityId,
-                accountId: results.get("b10")!.accountId,
-              },
+              // {
+              //   entityId: results.get("b1")!.entityId,
+              //   accountId: results.get("b1")!.accountId,
+              // },
+              // {
+              //   entityId: results.get("b9")!.entityId,
+              //   accountId: results.get("b9")!.accountId,
+              // },
+              // {
+              //   entityId: results.get("b11")!.entityId,
+              //   accountId: results.get("b11")!.accountId,
+              // },
+              // {
+              //   entityId: results.get("b2")!.entityId,
+              //   accountId: results.get("b2")!.accountId,
+              // },
+              // {
+              //   entityId: results.get("b3")!.entityId,
+              //   accountId: results.get("b3")!.accountId,
+              // },
+              // {
+              //   entityId: results.get("b10")!.entityId,
+              //   accountId: results.get("b10")!.accountId,
+              // },
               {
```

Comment:
?

Reply:
Reverted the comments and have created a new page to test the table block now: https://github.com/hashintel/hash/pull/188/files#diff-119d9d8e30c27fc2f6a023d9f971eb48a31f7541cf278bf1268a288be9a9a0acR842